cmd/main.go

```diff
@@ -104,6 +111,14 @@ func main() {
 	}
 }
 
+func buildLoadPricingProvider(cfg *config.Options, db *db.DB) (*pricing.NetworkLoadMultiplierCalculator, func(), error) {
+	calc := pricing.NewNetworkLoadMultiplierCalculator(
```

Comment:
Just return this back, then start and stop it in the main func? I feel like that would reduce confusion.